pallets/gated-marketplace/src/lib.rs

```diff
@@ -236,21 +240,21 @@ pub mod pallet {
 
 		#[transactional]
 		#[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
-		pub fn add_authority(origin: OriginFor<T>, author: T::AccountId, authority_type: MarketplaceAuthority, marketplace_id: [u8;32]) -> DispatchResult {
+		pub fn add_authority(origin: OriginFor<T>, account: T::AccountId, authority_type: MarketplaceAuthority, marketplace_id: [u8;32]) -> DispatchResult {
```

Comment:
Internal documentation needed for each extrinsic (the one which starts with `///`)